src/pushsource/_impl/model/ami.py

```diff
@@ -71,3 +71,39 @@ class AmiPushItem(VMIPushItem):
         validator=optional(instance_of(AmiBillingCodes)),
     )
     """Billing codes associated with this image."""
+
+    image_name = attr.ib(type=str, default=None, validator=optional_str)
+    """image name  """
+
+    description = attr.ib(type=str, default=None, validator=optional_str)
+    """A description for the image"""
+
+    arch = attr.ib(type=str, default=None, validator=optional_str)
+    """Architecture type of the image"""
+
+    virt_type = attr.ib(type=str, default=None, validator=optional_str)
+    """Virtualization type """
+
+    accounts = attr.ib(type=str, default=None, validator=optional_str)
+    """Accounts """
+
+    groups = attr.ib(type=str, default=None, validator=optional_str)
+    """Groups"""
+
+    release_notes = attr.ib(type=str, default=None, validator=optional_str)
+    """"Release notes for AMI """
+
+    operating_system_name = attr.ib(type=str, default=None, validator=optional_str)
+    """Name of operating system"""
```

Comment:
What is expected to be here, something like "RHEL" or just "Linux"?

Depending on it we can simplify by keeping the information just in [AmiRelease.product](https://github.com/release-engineering/pushsource/blob/master/src/pushsource/_impl/model/vms.py#L12-L14)?

Reply:
Ditto for this.